release/connect-streams.js

```diff
@@ -25837,7 +25837,7 @@
  
   var CCP_SYN_TIMEOUT = 1000; // 1 sec
   var CCP_ACK_TIMEOUT = 3000; // 3 sec
-  var CCP_LOAD_TIMEOUT = 3000; // 3 sec
+  var CCP_LOAD_TIMEOUT = 10000; // 10 sec
```

Comment:
not sure why this is showing as 10sec.

Reply:
Good catch, must not have npm run released.